File: UsbSerialConsole/app/src/main/java/jp/sugnakys/usbserialconsole/util/Log.java
package jp.sugnakys.usbserialconsole.util;

import jp.sugnakys.usbserialconsole.BuildConfig;

public class Log {

    public static void e(String tag, String msg) {
        android.util.Log.e(tag, msg);
    }

    public static void d(String tag, String msg) {
        if (BuildConfig.DEBUG) {
            android.util.Log.d(tag, msg);
        }
    }
}

File: UsbSerialConsole/app/src/main/java/jp/sugnakys/usbserialconsole/util/Log.kt
package jp.sugnakys.usbserialconsole.util

import android.util.Log
import jp.sugnakys.usbserialconsole.BuildConfig

object Log {
    @JvmStatic
    fun e(tag: String?, msg: String?) {
        Log.e(tag, msg!!)
    }

    @JvmStatic
    fun d(tag: String?, msg: String?) {
        if (BuildConfig.DEBUG) {
            Log.d(tag, msg!!)
        }
    }
}